Products/CMFPlone/browser/search.py

```diff
@@ -15,6 +15,7 @@
 from zope.i18nmessageid import MessageFactory
 from zope.publisher.browser import BrowserView
 from ZTUtils import make_query
+from zope.schema.interfaces import IVocabularyFactory
```

Comment:
Please put all imports in alphabetical order.

Reply:
done